src/components/Home/index.tsx

```diff
@@ -1,21 +1,28 @@
 import React, { useEffect } from "react";
+import { useNostoContext } from "../Provider/context.client";
 
 const NostoHome: React.FC = () => {
+
+  const { currentVariationData, countryLoadComplete } = useNostoContext();
+  
   useEffect(() => {
-    // @ts-ignore
-    window.nostojs((api) => {
-      api
-        .defaultSession()
-        .setResponseMode("HTML")
-        .viewFrontPage()
-        .setPlacements(api.placements.getPlacements())
-        .load()
-        .then((data: object) => {
-          // @ts-ignore
-          api.placements.injectCampaigns(data.recommendations);
-        });
-    });
-  }, []);
+    if (currentVariationData.currentVariation && countryLoadComplete.value) {
```

Comment:
We also need to support cases without multi currency. With the current approach Nosto won't render any recs when we don't pass the currentVariation